wiki/Do_you_really_want_to_ask_peppy/en.md

```diff
@@ -1,6 +1,6 @@
 # Do you really want to ask peppy?
 
-Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering forum PMs is quite hard to keep up with. He therefore asks that you do one of the following options:
+Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering the forum and DMs are quite hard to keep up with. He therefore asks that you do one of the following options:
```

Comment:
this is the page that is opened when you try to dm peppy in-game. not sure what "answering forums" means but it's not relevant (how do you "contact" peppy via the forums?)

```suggestion
Hi there! peppy understands your urge to want to contact him personally, but he asks that you first understand that this is **not** the place you want to ask him for help or support. He keeps osu! support separate from personal messages and also finds that answering DMs is quite hard to keep up with. He therefore asks that you do one of the following options:
```

Reply:
ah i dont know this wiki page is use for automaic peppy DM answer 💦
nice point, i'll fix